src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -22,49 +22,50 @@ public class HelpWindow extends UiPart<Stage> {
-            "4. FIND by NAME" + SINGLE_NEWLINE
-            + "Finds contacts whose names contain any of the given keywords." + SINGLE_NEWLINE
-            + "Format: find NAME" + SINGLE_NEWLINE
-            + "Example: find Jackson -> attempts to find a contact name containing 'Jackson'.";
-    public static final String FIND_ADDRESS_COMMAND_GUIDE =
-            "5. FIND by ADDRESS" + SINGLE_NEWLINE
-            + "Finds contacts whose address contain any of the given keywords." + SINGLE_NEWLINE
-            + "Format: find_add ADDRESS" + SINGLE_NEWLINE
-            + "Example: find_add geylang -> returns all contacts whose addresses contain 'geylang'.";
-
+            + "Example: delete 1 -> deletes the 1st contact, followed by undo -> add the contact back.";
+    public static final String FIND_COMMAND_GUIDE =
+            "4. FIND" + SINGLE_NEWLINE
+            + "Finds persons based on the given keywords." + SINGLE_NEWLINE
+            + "Format: find n/NAME [MORE NAMES]" + SINGLE_NEWLINE
+            + "             find a/ADDRESS [MORE ADDRESSES]" + SINGLE_NEWLINE
+            + "             find appt/APPOINTMENT DATE" + SINGLE_NEWLINE
+            + "Example: find n/John Alice -> finds all contacts whose names include 'John' and 'Alice'."
+            + SINGLE_NEWLINE
+            + "               find a/Tokyo Geylang -> finds all contacts whose address include 'Tokyo' and 'Geylang'."
+            + SINGLE_NEWLINE
+            + "               find appt/2024-01-01 -> finds all contacts whose appointment date matches '2024-01-01'.";
     public static final String EDIT_COMMAND_GUIDE =
-            "6. EDIT" + SINGLE_NEWLINE
+            "5. EDIT" + SINGLE_NEWLINE
             + "Edits an existing contact in the address book." + SINGLE_NEWLINE
             + "Format: edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [o/OCCUPATION] "
-            + "[a/ADDRESS] [t/TAG]…\u200B [appt/APPOINTMENTDATE]" + SINGLE_NEWLINE
+            + "[appt/APPOINTMENT_DATE] [a/ADDRESS] [t/TAG]…\u200B" + SINGLE_NEWLINE
             + "⚫Edits the person at the specified INDEX. " + SINGLE_NEWLINE
             + "⚫At least one of the optional fields must be provided." + SINGLE_NEWLINE
-            + "Example: edit 1 appt/ 2023-11-15 -> Edits the appointment date of the 1st person to be 2023-11-15.";
+            + "Example: edit 2 n/Betsy t/ -> Edits the name of the 2nd person to be 'Betsy' "
+            + "and clears all existing tags.";
 
     public static final String HELP_MESSAGE =
             ADD_COMMAND_GUIDE + DOUBLE_NEWLINE
             + DELETE_COMMAND_GUIDE + DOUBLE_NEWLINE
             + UNDO_COMMAND_GUIDE + DOUBLE_NEWLINE
-            + FIND_NAME_COMMAND_GUIDE + DOUBLE_NEWLINE
-            + FIND_ADDRESS_COMMAND_GUIDE + DOUBLE_NEWLINE
+            + FIND_COMMAND_GUIDE + DOUBLE_NEWLINE
             + EDIT_COMMAND_GUIDE + DOUBLE_NEWLINE
             + "For more details, please visit our website: " + USERGUIDE_URL;
 
```

Comment:
maybe you might want to add a note that mention advanced commands can be viewed in the user guide and the help window only shows basic commands?

Reply:
I believe this has been resolved. Other than that, LGTM! Good job!